StyleCop.Analyzers/StyleCop.Analyzers/OrderingRules/SA1212PropertyAccessorsMustFollowOrder.cs

```diff
@@ -84,17 +84,20 @@ private void HandlePropertyDeclaration(SyntaxNodeAnalysisContext context)
 
         private static void AnalyzeProperty(SyntaxNodeAnalysisContext context, BasePropertyDeclarationSyntax propertyDeclaration)
         {
-            var accessors = propertyDeclaration.AccessorList.Accessors;
-            if (propertyDeclaration.AccessorList.IsMissing ||
-                accessors.Count != 2)
+            if (propertyDeclaration?.AccessorList != null)
```

Comment:
:bulb: If you change this to the following, the diff for this method is simplified because it will not affect the indentation of the existing code.

```csharp
if (propertyDeclaration?.AccessorList == null)
    return;
```